src/components/Territory.vue

```diff
@@ -103,15 +103,22 @@ export default {
     ...mapActions({
       getTerritory: 'territory/getTerritory',
       resetNHRecords: 'territory/resetNHRecords',
+      checkinTerritory: 'territory/checkinTerritory',
     }),
 
     async resetNH() {
       if (window.confirm('Are you sure you want to reset NH records?')) {
         this.isLoading = true;
         await this.resetNHRecords(this.id);
         await this.getTerritory(this.id);
+        await this.checkinTerritory({
+          territoryId: this.id,
+          userId: this.user.id,
```

Comment:
UserId in this case is actually the person that checked out the territory. You can get that info from territory.status.publisher.id

To avoid undefined errors, use lodash get:

``
get(territory, 'status.publisher.id')
``